packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -70,12 +70,15 @@ function FontLibraryProvider( { children } ) {
 	} );
 
 	const libraryFonts =
-		( libraryPosts || [] ).map( ( post ) => {
-			post.font_family_settings.fontFace =
-				post?._embedded?.font_faces.map(
-					( face ) => face.font_face_settings
-				) || [];
-			return post.font_family_settings;
+		( libraryPosts || [] ).map( ( fontFamilyPost ) => {
+			return {
+				id: fontFamilyPost.id,
+				...fontFamilyPost.font_family_settings,
+				fontFace:
+					fontFamilyPost?._embedded?.font_faces.map(
+						( face ) => face.font_face_settings
+					) || [],
+			};
```

Comment:
I made a change to how the library fonts were processed to copy what is being done in getFontFamilyBySlug() which retains the .id property in the object so it doesn't need to be fetched later by slug just to get the id.

Reply:
Makes sense, thanks!